user/views.py

```diff
@@ -0,0 +1,87 @@
+from functools import partial
+from user.serializer import LoginSerializer, ProfileSerializer, UserSerializer
+from rest_framework import authentication, permissions
+from rest_framework.authtoken.models import Token
+from django.contrib.auth import authenticate
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from .utils import admin_required
+from rest_framework import status
+from user.models import Profile, User
+
+# Create your views here.
+
+class Userview(APIView):
+    authentication_classes = [authentication.TokenAuthentication]
+    permission_classes = [permissions.IsAuthenticated, admin_required]
```

Comment:
How the user will create an account if it is needed to authenticate first?

Reply:
fixed